ansible/roles/data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
+    version: main
+
+- name: CREATE THE USER
+  user:
+    name: "{{ USER }}"
+    state: present
+    home: "/home/{{ USER }}"
+    shell: /bin/bash
+
+- name: Ensure the locale exists
+  locale_gen:
+    name: en_US.UTF-8
+    state: present
+
+- name: set as default locale
+  command: locale-gen en_US.UTF-8
+
+- name: INSTALL THE PACKGES
+  apt:  
+    update_cache: yes
+    name: "{{ item }}"
+    state: present
+  loop:
+    - openjdk-8-jdk
+    - software-properties-common
+    - python3-pip
+    - python3.6-dev
+    - python3.6-venv
+    - python-virtualenv
+    - zip
+    - unzip
+     
+- name: ADD THE REPOSITORY
```

Comment:
do we have to add the repo before or after the package installation?
Use ansible ppa module for this.

Reply:
with the reference to the above comments i have added the repository before the installation package, regarding the usage the ppa module, i am not seeing universe repository under ppa module

Reply:
done